packages/language-core/lib/codegen/template/vFor.ts

```diff
@@ -51,7 +51,8 @@ export function* generateVFor(
 		ctx.addLocalVariable(varName);
 	}
 	let isFragment = true;
-	for (const argument of node.codegenNode?.children.arguments ?? []) {
+	// children is undefined with v-once (#4827)
+	for (const argument of node.codegenNode?.children?.arguments ?? []) {
```

Comment:
```suggestion
	for (const argument of (node.codegenNode?.children as CompilerDOM.ForRenderListExpression | undefined)?.arguments ?? []) {
```

Since the type of the children attribute is not nullable, the cast is added so that this change will not be automatically removed by `lint:fix`.